newjenkinsfile

```diff
@@ -0,0 +1,13 @@
+node('slave1') {
+  stage('checkout'){
+    checkout scm
+}
+  stage ('build'){
+  gradle = tool 'gradle4'
+      			sh "${gradle}/bin/gradle build"
+  }
+  stage ('post'){
```

Comment:
this won't run if the build fails
please make sure the badge is added also when the build fails